### DISCUSSION TITLE
Cómo quedará el comportamiento de los channels de un workflow desde el administrador

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#801] [2. Questions and Answers] [answered]
Hola a todos

**CÓMO ACCEDER AL WORKFLOW**
Revisando el adminitrador -> Workflow -> channels genera este link de referencia

<img width="592" alt="image" src="https://user-images.githubusercontent.com/61419471/160188169-95335cb7-9207-458f-994c-1508a07ca9da.png">

Url de ejemplo: https://web.cotalker.com/admin/workflowgroups/group_id/task
 
Me percaté que ahora se ven sólo las listas de tareas que tengan el flujo y al entrar en una de ellas se puede ver la información de la tasks, pero no opciones del canal en si.

<img width="567" alt="image" src="https://user-images.githubusercontent.com/61419471/160189218-d91c7105-8c97-462f-92f4-7539d80373d6.png">

al editar arroja esto

<img width="1163" alt="image" src="https://user-images.githubusercontent.com/61419471/160189263-c42dc156-a6ef-4437-952c-b370e444c019.png">

Por ende no podemos ir al canal de esa tasks para hacer: agregar properties, deshabilitar la escritura, cambiar de nombre, agregar usuarios, etc.

**CÓMO ACCEDER AL CHANNEL DE UN WORKFLOW**

La única forma en que uno puede acceder es:

https://web.cotalker.com/admin/groups/_id/channels

<img width="575" alt="image" src="https://user-images.githubusercontent.com/61419471/160190319-5a94d85b-03c6-4b94-bde9-969151383ba3.png">

**EN QUÉ AFECTA QUE NO SE PUEDE VER CHANNEL DE UN WORKFLOW**
1.- Que si en un workflow tiene un canal de conversación del área
2.- Ocultar los canales, cuando falla las tasks (los que salen con el nombre del formulario)

Entonces nos gustaría saber si será el comportamiento final que no se verá la información del canal dentro del administrador, tenerlo como en grupo para ver los canales  ó tendrá una sección aparte. 

@agallardol @acedward  nos pueden dar más detalle de esto?

gracias =>

**Comment by acedward** [comment, marked as answer]
Esto fue una decision de diseño. No vamos a permitir más canales sin tarea.
Pero vamos a dar soporte para que puedan modificar vía API los que ya existen.
Estamos trabajando para poder crear experiencias con canales estáticos y tareas simultáneamente.

**Reply by acedward** [reply]
Si necesitas soporte para algún canal **existente** déjanos saber!
